libs/net/models/Areas/Subscriber/Models/Content/ContentModel.cs

```diff
@@ -35,7 +35,7 @@ public class ContentModel : AuditColumnsModel
     /// <summary>
     /// get/set - The product.
     /// </summary>
-    public ProductModel? Product { get; set; }
+    public Product.ProductModel? Product { get; set; }
```

Comment:
Mixing models can lead to bugs.  It's generally better to create a copy of the model and place it in the convention location.  That way we know what models each controller is using.

Reply:
Good to know, will do